app/src/main/java/org/mozilla/tv/firefox/webrender/cursor/CursorModel.kt

```diff
@@ -0,0 +1,352 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.webrender.cursor
+
+import android.annotation.SuppressLint
+import android.graphics.PointF
+import android.view.KeyEvent
+import android.view.MotionEvent
+import androidx.annotation.CheckResult
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.Observables
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.PublishSubject
+import org.mozilla.tv.firefox.ScreenControllerStateMachine
+import org.mozilla.tv.firefox.ScreenControllerStateMachine.ActiveScreen.WEB_RENDER
+import org.mozilla.tv.firefox.ext.isUriYouTubeTV
+import org.mozilla.tv.firefox.framework.FrameworkRepo
+import org.mozilla.tv.firefox.session.SessionRepo
+import org.mozilla.tv.firefox.utils.Direction
+import org.mozilla.tv.firefox.utils.DirectionHelper
+
+// Constants that can be tweaked to adjust cursor behavior
+private const val BASE_SPEED = 5f
+private const val MAX_VELOCITY = 25f
+private const val MS_TO_MAX_ACCELERATION = 200
+private const val MAX_SCROLL_VELOCITY = 13
+// Other constants
+private const val MAX_ACCELERATION = MAX_VELOCITY - BASE_SPEED
```

Comment:
Perhaps it's poorly named.  Our acceleration ramps up as the user holds down the button, and this is as high as it can grow.  If that is true, this equation could not be changed without changing the meaning of either `MAX_VELOCITY` or `INITIAL_VELOCITY` (formerly `BASE_SPEED`).